### ISSUE TITLE
API lookup expressions stopped working

### ISSUE THREAD

**Issue by FestplattenSchnitzel** [netbox-community/netbox#17708]
### Deployment Type

Self-hosted

### NetBox Version

4.1.3

### Python Version

3.12.3

### Steps to Reproduce

Run e.g. https://demo.netbox.dev/api/dcim/interfaces/?type__n=virtual as described in https://demo.netbox.dev/static/docs/reference/filtering/#lookup-expressions .

### Expected Behavior

The query should return the results as described in https://demo.netbox.dev/static/docs/reference/filtering/#lookup-expressions .

### Observed Behavior

```
HTTP 400 Bad Request
Allow: GET, POST, PUT, PATCH, DELETE, HEAD, OPTIONS, TRACE
Content-Type: application/json
Vary: Accept

{
    "type__n": [
        "Select a valid choice. virtual is not one of the available choices."
    ]
}
```

**Comment by FestplattenSchnitzel** [comment]
Seems like (all?) other queries, e.g. https://netbox.agdsn.de/api/ipam/prefixes/?status__n=container , are broken too.

**Comment by arthanson** [comment]
@FestplattenSchnitzel can you please check the version number, this was just fixed in 4.1.3 so I want to double-check the version you are using.

**Comment by FestplattenSchnitzel** [comment]
We're definitely using Netbox version 4.1.3 from docker.io.  The deployment happened around 2024-10-03 15:30 UTC in case that's relevant.

Also, is this even relevant since the behaviour is reproducible on the demo instance (which reports version 4.1.3 as well)?
